randao/src/interfaces/IRandao.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import {IAxiomV1} from "axiom-contracts/contracts/interfaces/IAxiomV1.sol";
+
+interface IRandao {
+    function prevRandaos(uint32 blockNumber) external view returns (uint256);
+    function axiomAddress() external view returns (address);
+    function mergeBlock() external view returns (uint32);
+
+    function updateAxiomAddress(address _axiomAddress) external;

```

Comment:
not sure this function will be useful in the interface, since it's for the `onlyOwner` role

Reply:
That's a good point, I've moved it in my latest commit